sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_sender.py

```diff
@@ -0,0 +1,334 @@
+            message.schedule(schedule_time_utc)
+            message_data = {}
+            message_data['message-id'] = message.properties.message_id
+            if message.properties.group_id:
+                message_data['session-id'] = message.properties.group_id
+            if message.partition_key:
+                message_data['partition-key'] = message.partition_key
+            if message.via_partition_key:
+                message_data['via-partition-key'] = message.via_partition_key
+            message_data['message'] = bytearray(message.message.encode_message())
+            request_body['messages'].append(message_data)
+        return request_body
+
+
+class ServiceBusSender(BaseHandler, SenderMixin):
+    """The ServiceBusSender class defines a high level interface for
+    sending messages to the Azure Service Bus Queue or Topic.
+
+    :ivar str fully_qualified_namespace: The fully qualified host name for the Service Bus namespace.
+     The namespace format is: `<yournamespace>.servicebus.windows.net`.
+    :param ~azure.core.credentials.TokenCredential credential: The credential object used for authentication which
+     implements a particular interface for getting tokens. It accepts
+     :class:`ServiceBusSharedKeyCredential<azure.servicebus.ServiceBusSharedKeyCredential>`, or credential objects
+     generated by the azure-identity library and objects that implement the `get_token(self, *scopes)` method.
+    :keyword str queue_name: The path of specific Service Bus Queue the client connects to.
+    :keyword str topic_name: The path of specific Service Bus Topic the client connects to.
+    :keyword bool logging_enable: Whether to output network trace logs to the logger. Default is `False`.
+    :keyword int retry_total: The total number of attempts to redo a failed operation when an error occurs.
+     Default value is 3.
+    :keyword transport_type: The type of transport protocol that will be used for communicating with
+     the Service Bus service. Default is `TransportType.Amqp`.
+    :paramtype transport_type: ~azure.servicebus.TransportType
+    :keyword dict http_proxy: HTTP proxy settings. This must be a dictionary with the following
+     keys: `'proxy_hostname'` (str value) and `'proxy_port'` (int value).
+     Additionally the following keys may also be present: `'username', 'password'`.
+
+    .. admonition:: Example:
+
+        .. literalinclude:: ../samples/sync_samples/sample_code_servicebus.py
+            :start-after: [START create_servicebus_sender_sync]
+            :end-before: [END create_servicebus_sender_sync]
+            :language: python
+            :dedent: 4
+            :caption: Create a new instance of the ServiceBusSender.
+
+    """
+    def __init__(
+        self,
+        fully_qualified_namespace,
+        credential,
+        **kwargs
+    ):
+        # type: (str, TokenCredential, Any) -> None
+        if kwargs.get("from_connection_str", False):
```

Comment:
Not sure why we're passing this into the class kwargs? Seems like cheating...

Reply:
removed, I found it's not useful at all. thanks for calling out.